db/re/item_db.conf

```diff
@@ -61939,6 +61940,11 @@ item_db: (
 	Slots: 1
 	Loc: 256
 	View: 525
+	Script: <"
+		bonus bDex, 2;
+		skill TF_DOUBLE, 5;
```

Comment:
I don't know what to say Ridley. Sincerelly I don't think that have `skill TF_DOUBLE, max(5, getskilllv(TF_DOUBLE));` is the right way to understand `더 높은 레벨을 배우고 있을 시, 해당 레벨로 발동` since that text is also present (exactly) in Chick_Hat and it doesn't require to evaluate any skill learned, because it is implicit.

Even Sidewinder Card has that clausule on it's description:

> [더블어택] Lv 1
> 더블어택 스킬을 익힌 경우, 익힌 더블어택 스킬 레벨의 발동확률에 따른다.
> [Double Attack] Lv 1
> If you learn Double Attack skill, it will depend on the probability of playing Double Attack skill level.

And it does not make any check on player's skill level, just like Chick_Hat.

If after what I have explained you still can't share opinion, I can't, I won't and I will not push on this fix. I will take myself apart to allow other to send proper fix.